spec/features/services_spec.rb

```diff
@@ -193,5 +193,52 @@ def create_service(number)
         end
       end
     end
+
+    describe 'viewing service status' do
+      context 'when I have a service' do
+        let(:service) do
+          Service.create!(name: 'ABC Service', git_repo_url: 'https://github.com/some-org/some-repo.git',
+                          created_by_user: user)
+        end
+
+        context 'with no deployments' do
+          before do
+            visit "/services/#{service.slug}"
+          end
+
+          it 'there is no status' do
```

Comment:
really minor point - you should ideally be able to read `it '...'` lines out loud like a sentence, so could you change it to `it 'does not show a status'` or similar? thanks!